.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+# About email notifications, see <https://git.io/JUo27>.
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - 'master'
+  # TODO: fill the cron here
+  # schedule:
+  #   - cron: "30 0 * * 1" # Every Monday at half past midnight UTC skip-pr skip-stable
```

Comment:
Miri wants a daily cronjob, not just a weekly one.

Reply:
But what is the specific time ?

Reply:
It doesn't really matter... let's say 7:00 UTC?